v0_5/src/reset_checkpoint_types.rs

```diff
@@ -2,6 +2,37 @@ use elrond_wasm::elrond_codec::*;
 use elrond_wasm::BigUintApi;
 use elrond_wasm::Vec;
 
+/// Functions return this as status, if operation was completed or not.
+#[derive(PartialEq, Debug)]
+pub enum GlobalOperationStatus {
```

Comment:
A simple bool or int is enough. No need for enum serialize federalize and all.

Reply:
The way it is implemented, it doesn't incur any performance penalty.

Reply:
The serialization of this enum is currently identical to the serialization of the bool.

Reply:
How about the size of the generated wasm code ?